packages/react-components/src/organisms/LinkCard.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { Card, Headline3, Paragraph, Button } from '../atoms';
+import { noop } from '../utils';
+
+type LinkCardProps = {
+  readonly name: string;
+  readonly description: string;
+  readonly onClick?: () => void;
```

Comment:
I suspect that this would have an `href` instead? At least for modals I want to use routes for reason already stated. But can change it in the future

Reply:
Yeah, I'll do it in the modal PR 👍